AsyncDisplayKit/ASVideoPlayerNode.mm

```diff
@@ -502,6 +502,7 @@ - (void)showSpinner
       return spinnnerView;
     }];
     _spinnerNode.preferredFrameSize = CGSizeMake(44.0, 44.0);
+    _spinnerNode.backgroundColor = [UIColor clearColor];
```

Comment:
Because this method accesses the view at the bottom, we should add an Assert Main Thread at the top...

It would be slightly better to have this .backgroundColor inside the initWithViewBlock, not only because the spinner color itself is configured there, but because then the property bridge doesn't need to handle it (this is very very minor though :-P)